<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWorkerInfo</class>
 <widget class="QWidget" name="EditWorkerInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑信息</string>
  </property>
  <widget class="QLabel" name="worker_id_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>工号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="workerIDEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseWorkerBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="worker_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>选择职工</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_edit_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>580</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认所有修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkerNameEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="worker_name_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="worker_date_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>入职日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="worker_degree_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>学位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkerAreaEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="worker_area_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>现有住房面积</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkerAddressEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="worker_address_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_this_relative_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>470</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除这个亲属</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="relativeNameEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="relative_age_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="relative_appellation_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>称谓</string>
   </property>
  </widget>
  <widget class="QLabel" name="relative_name_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="relativeAgeEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>350</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="relativeAppellationEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="relative_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>选择亲属</string>
   </property>
  </widget>
  <widget class="QLabel" name="relative_gender_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseRelativeBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择该职工</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmRelativeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>200</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择该亲属</string>
   </property>
  </widget>
  <widget class="QLabel" name="worker_title_label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>职称</string>
   </property>
  </widget>
  <widget class="QLabel" name="alertLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="alertRelativelabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deletaWorkerButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除该职工</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="NewWorkerDatedateEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="WorkerDegreecomboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="WorkerTitlecomboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="RelativeGendercomboBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="birthdateEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
